tools/syz-repro/repro.go

```diff
@@ -23,8 +23,9 @@ var (
 	flagConfig = flag.String("config", "", "manager configuration file (manager.cfg)")
 	flagCount  = flag.Int("count", 0, "number of VMs to use (overrides config count param)")
 	flagDebug  = flag.Bool("debug", false, "print debug output")
-	flagOutput = flag.String("output", filepath.Join(".", "repro.txt"), "output description file (output.txt)")
+	flagOutput = flag.String("output", filepath.Join(".", "repro.txt"), "output syz repro file (output.txt)")
```

Comment:
while we are here: s/output.txt/repro.txt/

Reply:
Done. PTAL.